includes/frontend.php

```diff
@@ -142,6 +142,21 @@ class Frontend extends App {
 
 	private $google_fonts_index = 0;
 
+	/**
+	 * @var bool
+	 */
+	private $e_swiper_latest = false;
```

Comment:
Sure you need this as a class variable?

Reply:
@udidol I am using this in multiple functions in this file. So If I do not make this a class variable and set it once, then I will need to make calls to `Plugin::$instance->experiments->is_feature_active( 'e_swiper_latest' );` in multiple individual functions. What do you think?

Reply:
If it would have been used in a function that is called multiple times, caching would be a good idea I think.
Since it's only used in two places, both of which are only called once, and also since theoretically, experiment states could change via hooks etc. during runtime, I think it would be better practice to call `is_feature_active()` in both places instead of caching it.

Reply:
@udidol This was also done.